### PR TITLE
arpspoof: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/arpspoof.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# arpspoof
+
+> Forge ARP replies to intercept packets.
+> More information: <https://monkey.org/~dugsong/dsniff/>.
+
+- Poison all hosts to intercept packets on [i]nterface for host:
```
Comment (kbdharun) [inline pages/linux/arpspoof.md:6]:
```suggestion
- Poison all hosts to intercept packets on [i]nterface for the host:
```

File: pages/linux/arpspoof.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`sudo arpspoof -i {{wlan0}} {{host_ip}}`
+
+- Poison [t]arget to intercept packets on [i]nterface for host:
```
Comment (kbdharun) [inline pages/linux/arpspoof.md:10]:
```suggestion
- Poison [t]arget to intercept packets on [i]nterface for the host:
```

File: pages/linux/arpspoof.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`sudo arpspoof -i {{wlan0}} -t {{target_ip}} {{host_ip}}`
+
+- Poison both [t]arget and host to intercept packets on [i]nterface for host:
```
Comment (kbdharun) [inline pages/linux/arpspoof.md:14]:
```suggestion
- Poison both [t]arget and host to intercept packets on [i]nterface for the host:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

File: pages/linux/arpspoof.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# arpspoof
+
+> Forge ARP replies to intercept packets.
+> More information: <https://monkey.org/~dugsong/dsniff/>.
```
Comment (mfrw) [inline pages/linux/arpspoof.md:4]: nitpick:
```suggestion
> More information: <https://monkey.org/~dugsong/dsniff>.
```

**Review by einverne — APPROVED** [review]
Thank you for the new page, it looks good to me.

**Pull request merged**